### DISCUSSION TITLE
Nested documents can be used?

### DISCUSSION THREAD

**Discussion by masada-tea** [nuxt/content#1648] [Q&A]
Hi there, I'm now looking for way to use nested document in my project with nuxt/content v2.
For example, some articles about nuxt/content v2 using files like this...

- content
- - category-a
- - - articles-a
- - category-b
- - - articles-b

Is there a some way to use like this?
- content
- - category-a
- - category-a-index
- - - category-a1
- - - - articlea1
- - - categoriy-a2
- - category-b

For some visiblity, I want to use like this.

**Comment by ManasMadrecha** [comment]
Use it anyhow you like. Content Module doesn't limit how you structure your folders.
